src/sentry/runner/commands/cleanup.py

```diff
@@ -226,72 +226,52 @@ def is_filtered(model: type[Model]) -> bool:
 
             deletes = models_which_use_deletions_code_path()
 
-            # Track timing for each deletion stage to monitor execution progress and identify bottlenecks
-            with metrics.timer(
-                "cleanup.stage", instance=router, tags={"stage": "specialized_cleanups"}
```

Comment:
These stage metrics have not been that actionable. They have merely being interesting.

Hiding white spaces for this section of the PR makes the review easier.